Reason: Fix crossbuilding GIR
Upstream: No, not upstreamable since this is very specific to the Void setup
+++ clients/common/meson.build
@@ -35,7 +35,7 @@ if enable_introspection
     settings_docs,
     input: nm_property_docs,
     output: settings_docs,
-    command: [xsltproc, '--output', '@OUTPUT@', join_paths(meson.current_source_dir(), 'settings-docs.xsl'), '@INPUT@'],
+    command: [xsltproc, '--output', '@OUTPUT@', join_paths(meson.current_source_dir(), 'settings-docs.xsl'), join_paths(meson.current_source_dir(), '..', '..', '@INPUT@')],
   )

   test(
--- libnm/meson.build.orig	2020-12-28 16:03:30.641850261 +0100
+++ libnm/meson.build	2020-12-28 16:31:54.932136481 +0100
@@ -276,6 +276,14 @@
     'GI_TYPELIB_PATH=' + gi_typelib_path,
     'LD_LIBRARY_PATH=' + ld_library_path,
   ]
+  
+  if meson.is_cross_build()
+    qemu = '/usr/bin/qemu-@XBPS_TARGET_QEMU_MACHINE@-static'
+    qemu_python3_cmd = qemu + ' -L @XBPS_CROSS_BASE@ @XBPS_CROSS_BASE@/bin/python3'
+    generate_setting_docs = ['sh', '-c', qemu_python3_cmd, join_paths(meson.source_root(), 'tools', 'generate-docs-nm-settings-docs-gir.py')]
+  else
+    generate_setting_docs = [python.path(), join_paths(meson.source_root(), 'tools', 'generate-docs-nm-settings-docs-gir.py')]
+  endif
 
   nm_settings_docs_xml_gir = custom_target(
     'nm-settings-docs-gir.xml',
@@ -283,8 +291,7 @@
     output: 'nm-settings-docs-gir.xml',
     command: [
       generate_setting_docs_env,
-      python.path(),
-      join_paths(meson.source_root(), 'tools', 'generate-docs-nm-settings-docs-gir.py'),
+      generate_setting_docs,
       '--lib-path', meson.current_build_dir(),
       '--gir', '@INPUT@',
       '--output', '@OUTPUT@'
